src/vs/workbench/parts/debug/browser/debugEditorActions.ts

```diff
@@ -209,10 +211,97 @@ class ShowDebugHoverAction extends EditorAction {
 	}
 }
 
+class GoToBreakpointAction extends EditorAction {
+	constructor(private isNext, opts) {
+		super(opts);
+	}
+
+	public run(accessor: ServicesAccessor, editor: ICodeEditor, args: any): void | TPromise<void, any> {
+		const debugService = accessor.get(IDebugService);
+		const editorService = accessor.get(IEditorService);
```

Comment:
editor `Model` can be `null`

Reply:
It's not really checked in any other place in the file ;-)